features/performance/manifests/templates/02-machine-config-worker-rt-kargs.yaml.in

```diff
@@ -0,0 +1,26 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker-rt
+  name: worker-rt-kargs
+spec:
+  kernelArguments:
+  - 'default_hugepagesz=1G'
+  - 'hugepagesz=1G'
+  - 'hugepages=32'
+  - 'nohz=on'
+  - 'nosoftlockup'
+  - 'nmi_watchdog=0'
+  - 'audit=0'
+  - 'mce=off'
+  - 'kthread_cpus=0'
+  - 'irqaffinity=0'
+  - 'skew_tick=1'
+  - 'processor.max_cstate=1'
+  - 'idle=poll'
+  - 'intel_pstate=disable'
+  - 'intel_idle.max_cstate=0'
+  - 'intel_iommu=on'
+  - 'iommu=pt'
+  - 'isolcpus=${ISOLATED_CPUS}'
```

Comment:
Can you add also reserved_cpus=${RESERVED_CPUS}  ?  It would ease the calculations for https://github.com/openshift-kni/baremetal-deploy/pull/33 which is currently dependent on it. 
Im not sure if this is going to be added here or in https://github.com/openshift-kni/baremetal-deploy/pull/23

Reply:
@yanirq Sure, do we need to set it in both places, under the boot parameters, and under the kubelet config and should this parameters use the same CPU's?

Reply:
following our discussion its not given that reserved_cpus in terms of boot parameters and kubelet config but might be cut from it